homeassistant/components/tuya/cover.py

```diff
@@ -35,13 +35,29 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
     open_instruction_value: str = "open"
     close_instruction_value: str = "close"
     stop_instruction_value: str = "stop"
+    product_id: str | None = None
+    work_state: str | None = None
+    reverse: bool = True
+    set_position_open_close: bool = True
 
 
 COVERS: dict[str, tuple[TuyaCoverEntityDescription, ...]] = {
     # Curtain
     # Note: Multiple curtains isn't documented
     # https://developer.tuya.com/en/docs/iot/categorycl?id=Kaiuz1hnpo7df
     "cl": (
+        # AM43 Blind drive motor
+        # Note: Only product_id is "zah67ekd"
+        TuyaCoverEntityDescription(
+            key=DPCode.CONTROL,
+            product_id="zah67ekd",
+            current_position=DPCode.PERCENT_STATE,
+            set_position=DPCode.PERCENT_CONTROL,
+            device_class=CoverDeviceClass.BLIND,
+            work_state=DPCode.WORK_STATE,
+            reverse=False,
```

Comment:
If this is reversed, that would be an upstream bug that should be reported upstream and not be worked around here.